src/mavlink-router/endpoint.cpp

```diff
@@ -320,8 +320,8 @@ bool Endpoint::accept_msg(int target_sysid, int target_compid, uint8_t src_sysid
         }
     }
 
-    // This endpoint sent the message, and there's no other sys_comp_id: reject msg
-    if (has_sys_comp_id(src_sysid, src_compid) && _sys_comp_ids.size() == 1)
+    // This endpoint sent the message
```

Comment:
Minor nitpick: I'd just state on the comment that desired outcome is to reject the message.